modules/core/src/lib/deck.js

```diff
@@ -289,12 +291,12 @@ export default class Deck {
       mode: 'query',
       depth: 1
     });
-    this.stats.timeEnd('deck.pickObject');
+    this.stats.get('DeckGL: pickObject Time').timeEnd();
```

Comment:
I guess `Stats` knows it is a time since we use the `time*` functions, so we could make a note of that and autodeduce for presentation.

I think you see where I am getting with these comments...

How much to auto infer is sometimes down to personal taste.